cli/commands/console.go

```diff
@@ -30,11 +30,21 @@ func NewConsoleCmd(ec *cli.ExecutionContext) *cobra.Command {
 		Use:   "console",
 		Short: "Open console to manage database and try out APIs",
 		Long:  "Run a web server to serve Hasura Console for GraphQL Engine to manage database and build queries",
-		Example: `  # Start console:
+		Example: `
```

Comment:
Indentation is with tabs and spaces mixed here. 
Always use two spaces to indent multiline strings.